integration/simulation/simulation_tester.go

```diff
@@ -62,6 +63,23 @@ func testSimulation(t *testing.T, netw network.Network, params *params.SimParams
 	// execute the simulation
 	simulation.Start()
 
+	println("jjjj making call")
```

Comment:
Did you mean to leave these 'jjj's in? (Other one in the loop below)

Reply:
Nope, removed, sorry